aips/code-verification-api.md

```diff
@@ -0,0 +1,126 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Code Verification API
+author: 0xhsy
+discussions-to (*optional): <a url pointing to the official discussion thread>
+Status: Draft
+last-call-end-date (*optional): <mm/dd/yyyy the last date to leave feedbacks and reviews>
+type: Informational
+created: 5/22/2023
+updated (*optional): 5/23/2023
+requires (*optional): <AIP number(s)>
+---
+
+# AIP-X - Code Verification API
+  
+(Please give a temporary file name to your AIP when first drafting it, AIP manager will assign a number to it after reviewing)
+
+## Summary
+
+This API proposal seeks to introduce a standard protocol for the verification of Aptos Move smart contract code. This API is designed to ensure the safety and trustworthiness of smart contract code, assisting developers in effectively verifying their code. The standard suggests the common rules that should be followed by all parties building and operating smart contract code verification API servers within the Aptos network.
+
```

Comment:
Suggestion:

'Notice that this standard does not cover certification of provider of this API. Establishing trustworthiness of providers of this API is out of scope in this AIP, and left to community processes, possibly backed up by decentralized governance.'

Reply:
agreed.